SYCL/ESIMD/api/functional/ctors/ctor_fill.hpp

```diff
@@ -219,32 +219,45 @@ class run_test {
     });
     queue.wait_and_throw();
 
-    bool passed = true;
+    if (!are_bitwise_equal(result[0], base_value)) {
```

Comment:
```suggestion
    // Verify the base value was passed as-is
    if (!are_bitwise_equal(result[0], base_value)) {
```

Reply:
this comment was applied [here](https://github.com/intel/llvm-test-suite/pull/598/commits/04d6bab9c3fb63aeef04dbeea45f802d05edce82#diff-7fc90b589428e6e89076f07f5ff280a217b7f5c8e836bfe6c742734825074ef4R222)

Reply:
Minor, still looks a bit confusing to me:
```
if (!are_bitwise_equal(...)) {
  //  Verify ...
  passed = fail_test(...);
}

init_for_verification;
// Verify ...
for (..) {
  check_something;
}
```
Doesn't such comment placement mean the actual verification is made by `fail_test()`, not by `are_bitwise_equal()`? What would you say about using something like this instead?

```
//  Verify ...
if (!are_bitwise_equal(...)) {
  passed = fail_test(...);
}

// Verify ...
init_for_verification;
for (..) {
  check_something;
}
```
By doing so you might provide a comment to the block of code, visually and semantically separating them

Reply:
Minor, still looks a bit confusing to me:
```
if (!are_bitwise_equal(...)) {
  //  Verify ...
  passed = fail_test(...);
}

init_for_verification;
// Verify ...
for (..) {
  check_something;
}
```
Doesn't such comment placement mean the actual verification is made by `fail_test()`, not by `are_bitwise_equal()`? What would you say about using something like this instead?

```
//  Verify ...
if (!are_bitwise_equal(...)) {
  passed = fail_test(...);
}

// Verify ...
init_for_verification;
for (..) {
  check_something;
}
```
By doing so you might provide a comment to the block of code, visually and semantically separating them

Reply:
I've applied this comment in this commit: f4636f5c6bbda87420b54c9ceb07763559d30865